database/buildSchema/13_template_permission.sql

```diff
@@ -12,24 +12,24 @@ CREATE TABLE public.template_permission (
 
 CREATE VIEW permissions_all AS (
     SELECT
-        permission_policy.type AS "permissionType",
+        "user"."username" AS "username",
```

Comment:
`"username"` doesn't need to be in quotes, I think: `"user".username AS "username"`,